### ISSUE TITLE
Beatmaps not filtering properly(?) while having active supporter tag

### ISSUE THREAD

**Issue by Pinkfluke** [ppy/osu-web#8537]
Hi,

I just bought the supporter tag so that I can search beatmaps by rank achieved (on web). I have some S ranks I want to get to SS ranks. I have one S rank on osu!catch that I want to trace down. However, the search results don't show me anything despite having that S rank on my profile. I am sure I am selecting "Rank Achieved: S" upon search and I am also sure I am selecting osu!catch specifically. I triple checked and I'm doing it right, it is working for other maps. What's going on? 

I really hope you could look into this, since this is the one and only reason I got myself the supporter tag. And if in any case it is not possible to be fixed, could you send me the link to that one beatmap? 

My IGN is: Diglett
Here's the link to my profile: https://osu.ppy.sh/users/2090034

Kind regards

**Comment by nanaya** [comment]
Try enabling "include converted beatmaps" option.

**Comment by Pinkfluke** [comment]
I tried and that seemed to make it show up. However, the map apparently doesn't have osu!catch anymore, just the regular osu! mode. And so that means I won't ever be able to improve that one S rank. Is it still possible to improve it or maybe get my S rank removed? I seriously only got the supporter tag so that I could improve that one S rank.

Kind regards.

**Comment by nanaya** [comment]
Maybe actually check the converted maps ranking first...

**Comment by nanaya** [comment]
Also, even without doing that, as you don't have over 100 scores for catch, all of your scores are viewable directly from your profile page's Best Performance section.

**Comment by peppy** [comment]
It could also be an issue in miscalculation of your rank totals. I'll run a recalculation on your account in case.

**Comment by Pinkfluke** [comment]
Thanks for your replies. I indeed found out it is this beatmap I have the S rank on: https://osu.ppy.sh/beatmapsets/29290#fruits/104147

However, that beatmap doesn't have osu!catch beatmaps anymore. Maybe they got removed over time? I don't know, but that means I will never be able to improve the S rank I have on the map.

Is it one way or another still possible to improve it? Or maybe get my S rank removed? I'd do anything to turn it into an SS or get it removed.
